bigquery/tables.rb

```diff
@@ -169,10 +162,9 @@ def export_table_data_to_cloud_storage project_id:, dataset_id:, table_id:,
   # table_id     = "ID of the table to export file data from"
   # storage_path = "Storage path to export to, eg. gs://bucket/file.csv"
 
-  require "google/cloud"
+  require "google/cloud/bigquery"
 
-  gcloud   = Google::Cloud.new project_id
-  bigquery = gcloud.bigquery
+  bigquery = Google::Cloud::Bigquery.new
```

Comment:
project: project_id is necessary because this is inside a function which has a project_id parameter.